DOCS: update README (#219)
django-cms/djangocms-link: README.rst

    :target: https://www.djangoproject.com/
.. |djangocms| image:: https://img.shields.io/badge/django%20CMS-3.7%2B-blue.svg
    :target: https://www.django-cms.org/

Updating from `cmsplugin-filer <https://github.com/django-cms/cmsplugin-filer>`_
--------------------------------------------------------------------------------

> `cmsplugin-filer` was used a few times ago to create file, folder, image, link, teaser & video plugins on your DjangoCMS projects. Now the project is archived, and you might be wondering how to migrate your old instances without having to copy them manually to the new `djangocms-<file|picture|link|...>` plugins.

Reviewer: ```suggestion
`cmsplugin-filer` was used to create file, folder, image, link, teaser & video plugins on your django CMS projects. Now the project has been archived, and you might be wondering how to migrate your old instances without having to copy them manually to the new `djangocms-<file|picture|link|...>` plugins.
```

Reviewer: And if you could update the other PRs to match this, that would be wonderful. It's a minor change I know, but I feel it reads a little better. Thanks for your contribution 👍

Author: I fixed the content using your review and the one Fabian wrote on the djangocms-picture PR, thanks!
